test/util/IndexTestHelpers.cpp

```diff
@@ -124,7 +123,7 @@ Index makeTestIndex(const std::string& indexBasename,
   // these tests.
   static std::ostringstream ignoreLogStream;
   ad_utility::setGlobalLoggingStream(&ignoreLogStream);
-  std::string inputFilename = indexBasename + ".ttl";
+  std::string inputFilename = baseName + ".ttl";
```

Comment:
If you here change the baseName vocabulary to something that is different but derived from the `baseName`, then you immediately have a test for your new feature.
(make sure to also pass then then vocabularyBasename to the `getAllIndexFilenames` function above).